packer/ansible/roles/mongodb/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 - name: Install MongoDB
   apt: pkg={{ item }} state=installed
   with_items:
-    - mongodb=1:2.4.9-1ubuntu2
+    - mongodb=1:2.6.10-0ubuntu1
```

Comment:
what's the reason to switch to 2.6.10?

Reply:
Confirmed with @changev , we can use the default mongodb version in both 14.04 & 16.04 ubuntu.